content/blog/2016-01-26-one-year-of-open-prometheus-development.md

```diff
@@ -0,0 +1,139 @@
+usage within the company, as well as some early adoption by our friends at
+Docker and Boexever in 2014. Over the years, Prometheus was growing more and
+more mature and although it was already solving people's monitoring problems,
+it was still unknown to the wider public.
+
+## Going public
+
+Everything changed for us a year ago, in January of 2015. After more than two
+years of development and internal usage, we felt that Prometheus was ready for
+a wider audience and decided to go fully public with our official [announcement
+blog post](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud),
+a [website](http://prometheus.io/), and a series of
+[related](http://www.boxever.com/tags/monitoring)
+[posts](http://5pi.de/2015/01/26/monitor-docker-containers-with-prometheus/).
+We already received a good deal of attention during the first week after the
+announcement, but nothing could prepare us for what happened a week later:
+someone unknown to us (hello there,
+[jjwiseman](https://news.ycombinator.com/user?id=jjwiseman)!) had submitted
+[the Prometheus website](http://prometheus.io/) to Hacker News and somehow their
+post had made it to [all the way to the top](https://news.ycombinator.com/item?id=8995696).
+
+This is when things started going slightly crazy in a good way. We saw a sharp
+rise in contributors, mailing list questions, GitHub issues, IRC visitors,
+requests for conference and meetup talks, and increasing buzz on the net in
+general. Since the beginning, we have been very lucky about the quality of our
+newly gained community: The kind of people who were attracted to Prometheus
+also turned out to be very competent, constructive, and high-quality
+contributors and users. The ideal open-source scenario of receiving a lot of
+value back from the community was a reality pretty much from day one.
+
+What does all that Hacker News buzz look like in terms of GitHub stars? Try and
+see if you can find the exact moment in this graph (ironically, a Gnuplot and
+not Prometheus graph) when we went out of "dark mode" and got hit by Hacker
+News:
+
+[![Prometheus GitHub stars](/assets/prometheus_github_stars.png)](/assets/prometheus_github_stars.png)
+
+This attention also put us in the 4th place of GitHub's trending repositories
+worldwide:
+
+[![Prometheus trending on GitHub](/assets/prometheus_github_trending.png)](/assets/prometheus_github_trending.png)
+
+## After the first wave
+
+After those first weeks, the initial onslaught of incoming communication cooled
+down a bit, but we were and still are receiving constantly growing adoption.
+
+To give you an idea of the ecosystem, we now have:
+
+- 33 repositories managed as part of our GitHub organization
+- ~4800 total GitHub stars
+- >200 contributors
+- ~150 people in our IRC channel (`#prometheus` on [FreeNode](http://freenode.net/))
+- TODO: PR + commit stats
+- TODO: issue stats
+- TODO: conference talks
+- TODO: articles
```

Comment:
@brian-brazil and @beorn7 - you wanna help out with some of these? Brian, you already mentioned some talk and article stats?